browser/js/my-account/my-account.html

```diff
@@ -0,0 +1,79 @@
+<section id="account-page">
+	<div class="container">
+		<div class="row">
+			<div class="medium-4 large-4 columns">
+				<img class="thumbnail" height="250" width="250" src="http://dx3x.com/bills-progress/images/bill61_400x400.jpg">
+			</div>
+			<div class="medium-8 large-8 columns">
+				<div class="topOfPage">
+					<h3>{{ userSess.user.email }}</h3>
+					<h5>Your Account Info</h5>
+				</div>
+				<div class="row">
+					<div class="medium-6 large-12 columns">	
+						<p>Name: <input type="text" class="form-control" value="{{ userSess.user.fullName }}"></p>
+						<p>Address: 
+							<p>Street Address: <input type="text" class="form-control" value="Street Address"></p>
+							<p>City: <input type="text" class="form-control" value="City"></p>
+							<p>State: <input type="text" class="form-control" value="State"></p>
+							<p>Zip Code: <input type="text" class="form-control" value="Zip Code"></p>
+						</p>
+					</div>
+				</div>
+
+				<div class="row">
+					<div>
+						<a class="button medium expanded">Save Changes</a>
+					</div>
+				</div>
+				<!-- <div>
+					<a ng-click="showOrders(userSess.user.id)" class="button large expanded">SHOW ORDERS</a>
+				</div> -->
+			</div>
+		</div>
+		<hr>
+		<div class="column row">
+			<div class="tabs-content" data-tabs-content="example-tabs">
+				<div class="tabs-panel is-active" id="panel1">
+					<h4>Orders</h4>
+					<hr>
+					<div ng-repeat="order in orders" class="media-object stack-for-small">
+							<h5>Order Number: THE CONFIRMATION NUMBER</h5>
+							<div class="column row">
+								<div class="col-lg-6">
+									<p>Your Order Was Shipped To:</p>
+									<p>{{ order.address.lastName }}, {{ order.address.firstName }}</p>
+									<p>{{ order.address.address }}</p>
+									<p>{{ order.address.city }}, {{ order.address.state }}, {{ order.address.state }}</p>
+								</div>
+								<div class="col-lg-6">
+									<span ng-repeat="song in order.songs">
+										<p>SONGS ORDERED:</p>
+										<p>{{ song.title }}</p>
+										<button ng-click="submitReview(); reviewing = !reviewing" class="button small">Write a Review for {{ song.title }}</button>
+										<div ng-show="reviewing">
+											<label>
+												My Review
+												<p>STARS(Out of 5): <input type="text" ng-model="stars" style="width: 40px"></p>
+												<textarea ng-model="review" placeholder="None"></textarea>
+											</label>
+											<div class="column row">
+												<span class="col-lg-6 theReview"><button ng-click="reviewMade = true; submitReview()" class="button">Submit</button></span>
+												<span class="col-lg-6"><p ng-show="reviewMade">Review Submitted</p></span>
+											</div>
+										</div>
+									</span>
+								</div>
+							</div>
+							<p>Your Order Total: ${{ order.total }}</p>
+							<p>Is Your Order Shipped: {{ order.shipped }}</p>
+							<p>Date of Order: {{ order.createdAt }}</p>
+						<hr>
```

Comment:
Maybe we should also add <p>Order Delivered at: {{ order.deliveredAt || Not yet delivered }}</p>?